lib/client.js

```diff
@@ -40,6 +40,10 @@ function Client(opts) {
   this.retryTimeout = _.isUndefined(opts.retryTimeout) ? DEFAULT_RETRY_TIMEOUT : opts.retryTimeout;
   this.userAgent = _.isUndefined(opts.userAgent) ? util.format('%s/%s', package.name, package.version) : opts.userAgent;
 
+  if (opts.defaults) {
+    this.request = this.request.defaults(opts.defaults);
```

Comment:
This means that if you supply the `opts.defaults` you get a `request client wapper` but if you don't supply them you get a `request client`.